### ISSUE TITLE
"kfac.py"

### ISSUE THREAD

**Issue by mehdimashayekhi** [openai/baselines#603]
thanks much for sharing the baselines. out of all the implementations, the ```kfac.py``` is really hard to follow. I was wondering if you can add some more high level comments to this file. Thanks again

**Comment by EliasHasle** [comment]
There is also an implementation from the TensorFlow team here: https://github.com/tensorflow/kfac

Would it be an option for baselines to rely on their implementation instead?
